### DISCUSSION TITLE
Determine what makes a good “frigate” camera

### DISCUSSION THREAD

**Discussion by Sceptersax** [blakeblackshear/frigate#7508] [General]
I’m testing out several different cameras on frigate but I want to be able to tell what camera will be the most stable in the long run. Obviously I would need to just have them all record and see how long they continue to work but I’d like to know if there’s a better approach. I understand that the camera needs to be able to use ffmpeg and the ability to produce a rtsp stream but what else can I look for in the logs? What typically should I be looking for in a “good” camera for frigate? I was considering reolink but I’ve read somewhere they don’t play nice with frigate. I’d like to know what exactly to look for.

**Comment by NickM-27** [comment]
> I was considering reolink but I’ve read somewhere they don’t play nice with frigate. I’d like to know what exactly to look for.

They don't play nice period, you can find similar issues with Blue Iris and any other NVR software that isn't the reolink NVR

>  I’d like to know what exactly to look for.

The docs have examples: https://docs.frigate.video/frigate/hardware#cameras

After having all reolink cameras and now besides the reolink doorbell I have replaced them all with Loryta (dahua rebrand), Amcrest (less expensive dahua rebrand), and hikvision cameras. 

Frigate really isn't that particular about cameras, it's just following standard camera practice meaning cameras are best when they are wired, they need to follow the RTSP protocol well (ex: this is the reason reolink cameras have problems), and they should be a good camera in general for where they are being installed (good sensor for good vision at night, the appropriate resolution & field of view, etc.)

**Reply by Sceptersax** [reply]
I see. So really I’m just looking for cameras that follow good standard practices, have a large sensor… and are not reolink lol. Ok. Thanks Nick.

**Reply by NickM-27** [reply]
Yes, that would be my advice based on personal experience

**Reply by Sceptersax** [reply]
Hey Nick, been wanting to share this link with you for awhile. Just got around to doing so:  

https://www.thirtythreeforty.net/posts/2020/05/hacking-reolink-cameras-for-fun-and-profit/

Neolink.

**Reply by PrplHaz4** [reply]
Neolink work has been taken up at https://github.com/QuantumEntangledAndy/neolink with some very valuable fixes and updates, but you will be fighting the protocol the whole way.  RTSP is a standard and will work without any additional software in a number of different applications.

Unless you need battery or cellular cameras, you should stay away from Reolink.  Even then, you'll have to evaluate how it's going to be used, as battery and cellular are both problematic in Frigate because it needs constant reliable streams to be useful.
